frontend/ui/impl/src/test/java/com/virtuslab/gitmachete/frontend/ui/impl/GitMacheteErrorReportSubmitterTest.java

```diff
@@ -20,104 +18,80 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.mockito.Mockito;
 import org.powermock.api.mockito.PowerMockito;
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
 import org.powermock.reflect.Whitebox;
 
 @RunWith(PowerMockRunner.class)
-@PrepareForTest({GitMacheteErrorReportSubmitter.class, ApplicationInfo.class, PluginId.class, System.class, SystemUtils.class})
+@PrepareForTest({ApplicationInfo.class, PluginManagerCore.class, SystemUtils.class})
 public class GitMacheteErrorReportSubmitterTest {
 
-  // Used spy(...) here to still keep the original behaviour of the class and be able to stub getTemplateVariables method
-  // PowerMock API documentation Section 13 [https://javadoc.io/static/org.mockito/mockito-core/4.6.1/org/mockito/Mockito.html]
-  private static final GitMacheteErrorReportSubmitter reportSubmitter = PowerMockito
-      .spy(new GitMacheteErrorReportSubmitter());
+  private GitMacheteErrorReportSubmitter reportSubmitter;
   private static final LambdaLogger errorReportSubmitterLogger = PowerMockito.mock(LambdaLogger.class);
-  private static String defaultErrorReport;
-  private static String defaultErrorReportWithTemplate;
-  private static String errorReportWithStackTraceAndTitle;
-  private static String systemLineSeparator = "\n";
 
   @BeforeClass
-  @SneakyThrows
   public static void setUpStatic() {
-    defaultErrorReport = IOUtils.resourceToString("/error_report_uri.txt", StandardCharsets.UTF_8).strip();
-    defaultErrorReportWithTemplate = IOUtils.resourceToString("/error_report_uri_with_template.txt",
-        StandardCharsets.UTF_8).strip();
-    errorReportWithStackTraceAndTitle = IOUtils.resourceToString("/error_report_uri_with_stacktrace_title.txt",
-        StandardCharsets.UTF_8).strip();
-  }
-
-  @Before
-  public void setUp() {
     Whitebox.setInternalState(GitMacheteErrorReportSubmitter.class, errorReportSubmitterLogger);
```

Comment:
If we just want to supress the log output we can use a mock policy for slf4j that's ready out of the box in Power Mock - [Power Mock wiki](https://github.com/powermock/powermock/wiki/Mock-Policies)

Reply:
Okay, not true in this case. I think it's because we use **Lambda**Logger.

Reply:
Yeah, looks that a custom `PowerMockPolicy` would need to be implemented here... not worth the effort